src/commctl/client_script.py

```diff
@@ -255,7 +255,7 @@ def create_host(self, address, **kwargs):
         infile = kwargs['ssh-priv-key']
         b64_bytes = base64.b64encode(infile.read())
         data['ssh_priv_key'] = b64_bytes.decode()
-        if 'cluster' in kwargs:
+        if kwargs['cluster'] is not None:
```

Comment:
I think this is the relevant documentation: https://docs.python.org/2/library/argparse.html#default

> All optional arguments and some positional arguments may be omitted at the command line. The default keyword argument of add_argument(), whose value defaults to None, specifies what value should be used if the command-line argument is not present.